ext/mysqli/mysqli.c

```diff
@@ -744,15 +734,6 @@ PHP_MINIT_FUNCTION(mysqli)
 #endif
 #endif
 
-	REGISTER_LONG_CONSTANT("MYSQLI_SERVER_QUERY_NO_GOOD_INDEX_USED", SERVER_QUERY_NO_GOOD_INDEX_USED, CONST_CS | CONST_PERSISTENT);
-	REGISTER_LONG_CONSTANT("MYSQLI_SERVER_QUERY_NO_INDEX_USED", SERVER_QUERY_NO_INDEX_USED, CONST_CS | CONST_PERSISTENT);
-#ifdef SERVER_QUERY_WAS_SLOW
-	REGISTER_LONG_CONSTANT("MYSQLI_SERVER_QUERY_WAS_SLOW", SERVER_QUERY_WAS_SLOW, CONST_CS | CONST_PERSISTENT);
-#endif
-#ifdef SERVER_PS_OUT_PARAMS
-	REGISTER_LONG_CONSTANT("MYSQLI_SERVER_PS_OUT_PARAMS", SERVER_PS_OUT_PARAMS, CONST_CS | CONST_PERSISTENT);
-#endif
```

Comment:
Just to confirm, there is no way to access server_status from mysqli?

Reply:
I don't know of any way to use these constants. 

I was not aware that we can make constants deprecated. I will try to rewrite this PR now to make them deprecated instead of removing them.